### DISCUSSION TITLE
Stale shortcuts recommendation

### DISCUSSION THREAD

**Discussion by AustinConlon** [community/community#15856] [Mobile]
The last suggestion involves a repo that I'm no longer a part of. 

![image](https://user-images.githubusercontent.com/1305957/166123545-a167c9b9-9426-4520-89dc-f5bbec28c398.png)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.63.0 (66585841)
Device: iPhone14,3 (iOS 15.5)
</details>

**Comment by taki-on** [comment]
Thank you for sharing the issue! I will file it as a bug and we will investigate!
